lib/src/WebsocketControllersRouter.cc

```diff
@@ -184,16 +244,35 @@ void WebsocketControllersRouter::route(
     std::string wsKey = req->getHeaderBy("sec-websocket-key");
     if (!wsKey.empty())
     {
+        std::smatch result;
```

Comment:
Prefer putting this in its respective scope, within `if (iter == wsCtrlMap_.end())` block just before the for loop